src/test/java/com/aparapi/runtime/ArrayTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2016 - 2017 Syncleus, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.aparapi.runtime;
+
+import static org.junit.Assert.assertArrayEquals;
+
+import com.aparapi.Kernel;
+import java.util.Arrays;
+import org.junit.Test;
+
+
+public class ArrayTest {
+    @Test
+    public void test() {
+        VectorKernel b = new VectorKernel();
+        b.test();
+    }
+
+    public static class VectorKernel extends Kernel {
+        static final int size = 32;
+
+        static int[][] target = new int[size][size];
+
+        public VectorKernel() {
+            for (int i = 0; i < size; ++i) {
+                int[] ints = new int[size];
+                Arrays.fill(target[i], 99);
+
+                target[i] = ints;
```

Comment:
why are we filling `target[i]` array with value 99 then replacing that array with `ints`. ints is uninitalized and thus is filled with 0. The fill command has no effect here.

Reply:
yeah, typo here `Arrays.fill(target[i], 99);` ... fixed